riff-raff/app/views/deploy/form.scala.html

```diff
@@ -8,12 +8,22 @@
 
     <div class="clearfix"><p>&nbsp;</p></div>
 
+    <style type="text/css">
+        #projectInput_field > label::after {
+            content: '(exact match)';
```

Comment:
Is this where someone coming later would expect to find this string? Feels like this would be simpler added to the template directly, instead of splitting the content across the HTML and the CSS.

It's a minor point though, up to you.

Reply:
as discussed in person, because the input box and label is not directly but rather via a couple of layers of indirection , see https://github.com/guardian/riff-raff/blob/62817fe1a1d58bc27f40e779a24efe0e75530521/riff-raff/app/views/deploy/form.scala.html#L26 
...it was cleanest to add the 'secondary label' via pseudo `::after` element